src/vehicle/vehicle.cpp

```diff
@@ -212,7 +212,6 @@ namespace CityFlow {
         // collision free
         double v = getNoCollisionSpeed(leader->getSpeed(), leader->getMaxNegAcc(), vehicleInfo.speed,
                                        vehicleInfo.maxNegAcc, controllerInfo.gap, interval, 0);
-        int cnt = 0;
 
 //        if (v == -100 || v < vehicleInfo.speed - vehicleInfo.maxNegAcc * interval - 1e-7 ) return -200;
```

Comment:
Please remove the useless code. I know it is not your fault, but could you give us a hand? Thank you!